ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -512,6 +535,31 @@ class ServicesHomeScreen extends React.Component<Props, State> {
     );
   }
 
+  private renderLocalQuickSectionDeletion = (section: ServicesSectionState) => {
+    const onPressItem = () => {
+      if (this.props.userMetadata && this.props.selectedOrganizations) {
```

Comment:
```suggestion
      if (this.state.isWaitingForToast && this.props.userMetadata && this.props.selectedOrganizations) {
```
We could schedule one request a time

Reply:
Within https://github.com/teamdigitale/io-app/pull/1330/commits/98f43b5b9cb3c22ce16fccb1a5065e5cd41fc977 I tried to find an alternative solution, avoiding:
- the userMetadata upsert being disabled until a previous request has been sent and
- the usage of a component stae to manage the visibility of the toast message.

From my point of view, it could be more aligned to the 'optimistic update' approach. 
What do you think about it?

Reply:
@matteodesanti I see the toast content was wrong. It is updated by https://github.com/teamdigitale/io-app/pull/1330/commits/a421f67e05ab61dc55b445fc660f4ea42850807b

Reply:
@valentinacala This solution is even better: indeed it solves also the feedback about saving areas of interest with no connection.

<img src="https://user-images.githubusercontent.com/822471/66197872-8eb42200-e69b-11e9-9525-2a27e21a5f53.gif" height="600" />

Reply:
@valentinacala Did you do all changes? In that case I will merge this PR

Reply:
I completed changes to the code but I'm not sure about [this](https://github.com/teamdigitale/io-app/pull/1330#issuecomment-538023927) change. A feedback from @matteodesanti could be better before the merge.

Reply:
@valentinacala Sure! We'll wait!